components/atom/tag/src/index.scss

```diff
@@ -199,12 +199,37 @@ $base-class: '.sui-AtomTag';
     &--#{$name} {
       background-color: $bgc;
       color: $c;
+      fill: $c;
+      &#{$self}-actionable {
+        background-color: $bgc;
+        color: $c;
+        fill: $c;
+        &#{$self}--disabled {
+          cursor: default;
```

Comment:
Can't you move this selector outside the each?

Reply:
It will be compiled for each tag type :/

Reply:
Cannot. The not(disable) pseudo selector outside this @each gets a more restrictive css rule to the typed prop class generated by react-component.